datafusion/src/physical_plan/file_format/parquet.rs

```diff
@@ -250,7 +250,7 @@ impl ExecutionPlan for ParquetExec {
                 limit,
                 partition_col_proj,
             ) {
-                println!(
+                info!(
```

Comment:
What would you think about using `warn!` or error!` here instead of `info!`?

Reply:
+1, `warn!` maybe better.

Reply:
> What would you think about using `warn!` or error!`here instead of`info!`?

It's all acceptable to me except println